licenses/MIT.txt

```diff
@@ -0,0 +1,20 @@
+MIT License
```

Comment:
I'm not sure I follow the thinking about adding this new license file. Yes, we added a file that uses the MIT license, but could we not just add it to the single file that uses it?

Reply:
It's not necessary but it makes it far easier to communicate the licenses used by the project. Otherwise it's necessary to inspect every file in the project to see how they are licensed.